tests/unit/conftest.py

```diff
@@ -0,0 +1,97 @@
+import pytest
+
+from didcomm.common.types import DID, DID_URL
+from didcomm.did_doc.did_doc import DIDDoc, DIDCommService
+from didcomm.pack_encrypted import (
+    PackEncryptedConfig,
+    PackEncryptedParameters,
+)
+from didcomm.common.resolvers import ResolversConfig
+
+from tests import mock_module
+
+
+# ==============
+# unittest mocks
+# ==============
+
+
+@pytest.fixture
+def did() -> DID:
```

Comment:
I think that fixtures are used by Forward-related tests only, so do we really need to move them to top level conftest? Why don't keep it in forward-related packages only?